benefits/core/templates/core/landing.html

```diff
@@ -10,6 +11,11 @@
   {% endblock title %}
 {% endblock page-title %}
 
+{% block preload %}
+  <link rel="preload" as="image" href="{% static "img/benefits-bg-mobile.webp" %}" media="(max-width: 992px)" />
+  <link rel="preload" as="image" href="{% static "img/benefits-bg-desktop.webp" %}" media="(min-width: 992px)">
+{% endblock preload %}
```

Comment:
Just a question... if a screen is exactly `992px` wide, which one wins / which one pre-loads? Do they both?

Since the CSS is "mobile first", it seems like once we hit `992px` the variable is overridden to the desktop version.

Not a blocker at all, just curious if you knew.

Reply:
This was a great catch. I hadn't thought about what happens at exactly 992px.

I tried it out myself and saw that:
- in Firefox, only `benefits-bg-desktop.webp` is requested
- in Chrome, both `benefits-bg-desktop.webp` and `benefits-bg-mobile.webp` are requested
- in Edge, both `benefits-bg-desktop.webp` and `benefits-bg-mobile.webp` are requested
   - (I asked a family member to help me test on their Windows computer)

Some screenshots:

<details><summary>Firefox</summary>

![firefox](https://github.com/user-attachments/assets/1933fb70-55d9-4d2b-a463-84bb5cd5b438)

</details>

<details>
<summary>Chrome</summary>

![chrome](https://github.com/user-attachments/assets/f61c491a-6cbc-45c7-8b69-868120941643)

</details>

I'm seeing from [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries?trk=public_post_comment-text#targeting_media_types) that `max-width` and `min-width` are inclusive, so I think I'll change one of them to not include the comparison value. Thanks for catching!

Reply:
Fixed in be5a45d

<details><summary>Screenshot of network requests in Chrome at 992px:</summary>

![image](https://github.com/user-attachments/assets/52fb3991-81fd-42d9-960c-c638c1ca6636)

</details>